src/components/base-address/BaseAddress.vue

```diff
@@ -218,6 +219,12 @@ export default class BaseAddress extends Mixins(ValidationMixin, CountriesProvin
   @Prop({ default: false })
   readonly noPoBox: boolean
 
+  @Prop({ default: '' })
+  readonly deliveryInstructionsText: string
+
+  @Prop({ default: false })
+  readonly excludeBC: boolean
```

Comment:
Prop to exclude BC from province list

Reply:
Why do you need to do this? The UI design says the address must be in BC.

Reply:
Mihai mentioned in the meeting that it should be outside BC as per the requirement.  I will confirm again with Yui once he is back on Tuesday.

Reply:
Yes, to be confirmed next week.

Have a great weekend!

Reply:
Confirmed - Location should not be in BC. Hide BC from Province list. https://preview.uxpin.com/c32b7a6fabe9dea89e09a5ae63323f485a02396c#/pages/164499138/comment/sitemap